recipes/secapr/meta.yaml

```diff
@@ -1,27 +1,28 @@
-# build with: conda build --python 2.7 .
+{% set version = "1.1.0" %}
 
 package:
   name: secapr
-  version: '1.0.2'
+  version: {{ version }}
 
-about:
-  home: 'https://github.com/AntonelliLab/seqcap_processor'
-  license: MIT
-  summary: 'Process sequence-capture FASTQ files into alignments for phylogenetic analyses. Integrates allele phasing, producing haplotype alignments.'
 source:
-  url: https://github.com/AntonelliLab/seqcap_processor/archive/v1.0.2.tar.gz
-  sha256: ada58295f1d02ca2da0b57a49d74770afcb3ac3137c25c32ecf19bb44937dbfa
+  fn: secapr_{{ version }}.tar.gz
+  url: https://github.com/AntonelliLab/seqcap_processor/archive/v{{ version }}.tar.gz
+  sha256: 988a37eaf9fc050f482c0ae1acb9994ce144a04d2de9ae1b827469398e7bb5dd
 
 build:
   skip: True  # [not py27]
   number: 0
-  script: $PYTHON setup.py install
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - secapr = secapr.__main__:main
 
 requirements:
   build:
     - python
+    - setuptools
     - biopython
     - pandas
```

Comment:
```diff
-    - biopython
-    - pandas
```
`setup.py` is nice and doesn't import anything apart from `sys`, `setuptools` and `versioneer` 😄